refarch-backend/pom.xml

```diff
@@ -36,6 +36,7 @@
         <sonar-maven-plugin.version>3.9.0.2155</sonar-maven-plugin.version>
         <pmd-maven-plugin.version>3.23.0</pmd-maven-plugin.version>
         <spotbugs-maven-plugin.version>4.8.6.2</spotbugs-maven-plugin.version>
+        <spotbugs-annotations.version>4.8.6</spotbugs-annotations.version>
```

Comment:
Must this match the version of `spotbugs-maven-plugin`? (at least until X.X.X patch level). If yes I'd add a comment after `<spotbugs-annotations.version>` which indicates it has to match

Reply:
they dont need to match. I tried to run it with an older version of spotbugs-annotations and it works. Also they never released on the same day. Spotbugs always released a few days earlier then the annotations. I would not make a comment and just let renvote handle it.

Reply:
Ok thats fine. :)